### PR TITLE
Update Fedora Docker files from master branches in dist-git.

### PR DESCRIPTION
**Pull request by frenzymadness**
Master branch should have the latest images.

### CODE REVIEW

**Comment by frenzymadness** [comment]
I have no idea what happened on RHEL8 when I change only Fedora images.

**Comment by frenzymadness** [comment]
[test]

**Comment by frenzymadness** [comment]
More general error, great.
```
+ rsync -avzP -e 'ssh -F ssh_config' /var/lib/jenkins/workspace/rhscl-images-s2i-rh-rhel8/ host:sources
ssh: connect to host 10.0.151.47 port 22: No route to host
rsync: connection unexpectedly closed (0 bytes received so far) [sender]
rsync error: unexplained error (code 255) at io.c(226) [sender=3.1.2]
```

**Comment by frenzymadness** [comment]
Looking for help here @phracek @pkubatrh

**Comment by pkubatrh** [comment]
This looks like some CI mishap. Going to try rerunning the job

**Comment by frenzymadness** [comment]
[test]

**Comment by frenzymadness** [comment]
[test-openshift]

**Comment by frenzymadness** [comment]
I have no idea what is happening there:
```
14:05:42 + docker run -d --rm --cidfile=/tmp/tmp.icphybhmHr/cid centos/ruby-26-centos7-testapp
14:05:42 Unable to find image 'centos/ruby-26-centos7-testapp:latest' locally
14:05:42 Trying to pull repository docker.io/centos/ruby-26-centos7-testapp ... 
14:05:42 /usr/bin/docker-current: repository docker.io/centos/ruby-26-centos7-testapp not found: does not exist or no pull access.
```

**Comment by frenzymadness** [comment]
[test]

**Comment by pkubatrh** [comment]
Looks like a bug that will eventually be fixed by https://github.com/sclorg/s2i-ruby-container/pull/247

rhel7 specific, so we can ignore the failure for this PR

File: core/Dockerfile.fedora [hunk 1]
```diff
@@ -41,7 +41,6 @@ RUN INSTALL_PKGS="bsdtar \
   findutils \
   gettext \
   groff-base \
-  rsync \
```
Comment (pkubatrh) [inline core/Dockerfile.fedora]: Why is rsync getting removed here?

Comment (frenzymadness) [reply core/Dockerfile.fedora]: Well, rsync was added here in https://github.com/sclorg/s2i-base-container/commit/808085fd115a0f96049b0f5d7ad12b953d231af0 but the change was never synced down to dist git. Should I sync these repositories in the other way around Github → dist git? The current approach has a benefit that we'll have here the sources of currently available images.

**Comment by frenzymadness** [comment]
Ok, this repo is the only point of truth now and we'll sync only changes from here to dist git.

**Pull request closed without merging**